Lib/glyphsLib/builder/builders.py

```diff
@@ -492,6 +493,13 @@ def _apply_bracket_layers(self):
                 )
                 self._designspace.addRule(rule)
 
+        # Set feature for rules
+        feat = self.font.customParameters["Feature for Feature Variations"]
+        if feat == "rclt":
+            self._designspace.rulesProcessingLast = True
+        elif feat and feat != "rvrn":
+            self._designspace.lib[FEAVAR_FEATURETAG_LIB_KEY] = feat
```

Comment:
how about always using the FEAVAR_FEATURETAG_LIB_KEY and forget about the rulesProcessingLast?

Reply:
when the lib key is set, the `processing` attribute is ignored (it's always implicitly "last" for anything but 'rvrn' which is special and always applied first).